ldes-server-infra-mongo/mongock-changeset-3/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/mongock/changeset3/entities/MemberSequenceEntityV1.java

```diff
@@ -0,0 +1,33 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset3.entities;
```

Comment:
overbodig ?

Reply:
No, the class is needed to give all the old members an index :-)